requirements.txt & macros plugin

oomIRL opened the pull request:
- python pkgs install uses requirements.txt
- adding mkdocs macros plugin
- updating tag v0.1.1

oomIRL committed:
switch python pkgs install to requirements.txt

oomIRL committed:
Adding mkdocs macros

oomIRL committed:
Merge pull request #8 from bcgov/DEVX1300-requirements-txt

switch python pkgs install to requirements.txt

oomIRL committed:
Updated action.yml to use a new Docker image with tag 'v0.1.1'.

sheaphillips commented:
we should hold off on merging this as there seems to be an issue with updating existing content that came up somewhere along the line. doing some troubleshooting now.

oomIRL commented:
Hey @sheaphillips 
Curious if there are any updates on this one?

sheaphillips committed:
DEVX-904

- add htmltest for link validation.
- update index.md to describe use of htmltest

also:
- lock down some versions in requirements.txt

sheaphillips committed:
Merge pull request #10 from bcgov/DEVX-904

DEVX-904

sheaphillips committed:
add config to workflow file to build Docker image for amd64 and ARM architectures.

sheaphillips committed:
add config to workflow file to build Docker image for amd64 and ARM architectures.

sheaphillips committed:
add config to workflow file to build Docker image for amd64 and ARM architectures.

sheaphillips committed:
Updated action.yml to use a new Docker image with tag 'v0.2.0'.
